api/models/orderDao.js

```diff
@@ -0,0 +1,103 @@
+const dataSource = require("./dataSource");
+const bidDao = require("./bidDao");
+const itemDao = require("./itemDao");
```

Comment:
```suggestion
const {
  dataSource,
  bidDao,
  itemDao
} = require("./index");
```
우리는 index.js를 활용하고 있기 때문에 위와 같이 사용하는게 보다 관리하기 용이할 것입니다.